src/cr/cube/cube.py

```diff
@@ -735,7 +735,7 @@ def valid_counts(self):
 
     @lazyproperty
     def _flat_values(self):  # pragma: no cover
-        """Return tuple of mean values as found in cube response.
+        """Return ndarray of np.float64 values as found in cube response.
```

Comment:
It's my habit, and perhaps more proper, to leave out the "Return " suffix on docstrings of properties (lazy or otherwise) since they behave like attributes. So we document a property like we would an attribute, by stating what it _is_ rather that what it "returns", since returning a value is a characteristic of a function and attributes just "are" their value.

As a side benefit, it opens up first-docstring-line real estate for more descriptive words.